### PR TITLE
7 vote on a review

### PR DESCRIPTION
**Pull request by HassanAlhaji**

### CODE REVIEW

File: src/components/SingleReview.jsx [hunk 1]
```diff
@@ -1,14 +1,45 @@
...
 function SingleReview() {
   const [review, setReview] = useState();
   const [isLoading, setIsLoading] = useState(true);
   const [err, setErr] = useState(null);
+  const [upvote, setUpvote] = useState(false);
+  const [downvote, setDownvote] = useState(false);
   const { reviewId } = useParams();
 
+  const upVoteHandler = () => {
+    console.log("you clicked the upvote button!");
```
Comment (B7Bprog) [inline src/components/SingleReview.jsx]: Please remove all logs.

File: src/components/SingleReview.jsx [hunk 2]
```diff
@@ -1,14 +1,45 @@
...
-import CommentCard from "./CommentCard";
+import { getSingleReview, updateReviewVotes } from "../utils/utils";
+import { BiUpArrow, BiDownArrow } from "react-icons/bi";
+import ReviewComments from "./ReviewComments";
+import "./SingleReview.css";
 
 function SingleReview() {
   const [review, setReview] = useState();
   const [isLoading, setIsLoading] = useState(true);
   const [err, setErr] = useState(null);
```
Comment (B7Bprog) [inline src/components/SingleReview.jsx:11]: You can make use of this to visualize any errors.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Hassan, I like the functionality of your votes and the optimistic rendering. Please implement error handling so when a patch request goes wrong you revert the votes to the original and display an error message for the user.

When you're done with this, please merge.

File: src/components/SingleReview.jsx (hunk 1 above)
Comment (HassanAlhaji) [reply src/components/SingleReview.jsx]: Done.

**Pull request merged**